server/index.js

```diff
@@ -11,11 +12,17 @@ const PORT = 3000;
 // use body parser on incoming requests
 app.use(bodyParser.json());
 
-// write dummy response to GET req
-app.get('/', (req, res) => {
+// serve up static html file
+app.use('/', express.static('public'));
```

Comment:
Note, for my module to work, because it relies on the id passed through in the URL, the URL is `/:listingid` instead of `/`. I think this should work with everyone else's modules

Reply:
or actually, you may want to remove '/' entirely (that works for me locally with multiple modules)